src/modules/dashboard/components/ColonyHome/ColonyFunding/ColonyFunding.tsx

```diff
@@ -33,11 +30,9 @@ interface Props {
 const displayName = 'dashboard.ColonyHome.ColonyFunding';
 
 const ColonyFunding = ({ colony, currentDomainId }: Props) => {
-  const { walletAddress, ethereal, username } = useLoggedInUser();
+  const { ethereal, username } = useLoggedInUser();
 
-  const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
-
-  const canMoveFunds = !!username && !ethereal && canFund(allUserRoles);
+  const canMoveFunds = !!username && !ethereal;
 
   const {
     colonyAddress,
```

Comment:
Line 60 below `canMoveFunds ? ...` is the logic that needs to be removed.

Reply:
Got it! Thanks Will.